pkg/storage/stores/shipper/indexshipper/tsdb/multi_file_index.go

```diff
@@ -115,7 +115,7 @@ func (i *MultiIndex) forMatchingIndices(ctx context.Context, from, through model
 	queryBounds := newBounds(from, through)
 
 	return i.iter.For(ctx, i.maxParallel, func(ctx context.Context, idx Index) error {
-		if Overlap(queryBounds, idx) {
+		if Overlap(idx, queryBounds) {
```

Comment:
q: wondering if any other places of `Overlap` usage need this swap of arguments as well?

Reply:
I checked, but all other places already use the Overlap function with the correct argument order.